### PR TITLE
Navbar finished and moved into application erb file. Needs logic and styling.

### PR DESCRIPTION
**Pull request by karimabuseer**

### CODE REVIEW

File: app/views/layouts/navbar.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="../../assets/stylesheets/navbar.css">
+  </head>
+  <body>
+
+      <ul>
+        <li class="nav-item">
+          <!-- <a class="navbar-brand" href="path-to-root"><img src="path-to-logo"></a> -->
```
Comment (pav0107) [inline app/views/layouts/navbar.html]: You can remove this file as it's a duplication of application.html.erb content

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,14 +1,37 @@
...
 
     <%= stylesheet_link_tag    'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
   </head>
 
   <body>
+   <div class="navbar">
+      <ul class="nav-list">
+        <li class="nav-item"> 
+          <!-- <a class="navbar-brand" href="path-to-root"><img src="path-to-logo"></a> -->
```
Comment (pav0107) [inline app/views/layouts/application.html.erb]: You could remove this line.

**Review by pav0107 — CHANGES_REQUESTED** [review]

File: app/views/layouts/navbar.html (hunk 1 above)
Comment (karimabuseer) [reply app/views/layouts/navbar.html]: Done

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (karimabuseer) [reply app/views/layouts/application.html.erb]: Done

**Pull request merged**